astropy/wcs/wcsapi/fitswcs.py

```diff
@@ -553,7 +554,7 @@ def _get_components_and_classes(self):
                 try:
                     target.transform_to(ICRS())
                 except Exception:
-                    warnings.warn(
+                    log.debug(
```

Comment:
This is a behavior change, so probably need a change log?

Reply:
Happy to add one

Reply:
Done, I am not sure if the message is useful or if its the correct category though.